core/app/models/spree/property.rb

```diff
@@ -44,11 +43,5 @@ def with_uniq_values_cache_key(product_properties_scope, &block)
       uniq_values_cache_key = ['property-uniq-values', cache_key_with_version]
       Rails.cache.fetch(uniq_values_cache_key) { block.call }
     end
-
-    def ensure_product_properties_have_filter_params
```

Comment:
@aleksandarpetrushev why are we removing this?

Reply:
https://github.com/spree/spree/pull/11567/files/4223de53240f06d580736592d0a679d97af8bcaf#diff-f26177b5df7f418dc85dd8c4ea874b6289b73cf36654b241f57fd4ee74109577L51 didn't notice the .not there